buddyboss/buddyboss-platform
# #2933 - Fix: Forum page with group associated title tag issue

### All Submissions:






### Changes proposed in this Pull Request:



Fixes #2933 .

### How to test the changes in this Pull Request:

1. Create a forum and associate it with the group.
2. Check the forum page.
3. You'll see that title gets like Discussion - Group - Site Name.

Related Issue solved:
1. Copy the forum with no group and discussion URL
2. Paste in the activity feed
3. Can see the live preview is working fine instead of https://prnt.sc/1rtktn4

### Proof Screenshots or Video
https://vimeo.com/645160371/e987c33080


### Other information:




### Changelog entry

> Enter a summary of all changes on this Pull Request. This will appear in the changelog if accepted.
